README.md

```diff
@@ -107,6 +107,30 @@ config :k8s,
 
 This will add a cluster named `default`. When no configuration information is provided, the `k8s` library will use the service account of the pod.
 
+## Running outside of a cluster
+
+Running an operator outside of kubernetes is not recommended for production use, but can be very useful when testing.
+
+To start your operator and connect it to an existing cluster, one must first:
+1. Have configured your operator. The above example is a good place to start.
+2. Have some way of connecting to your cluster. The most common is to connect using your kubeconfig as in the example:
+```elixir
+# config.exs
+config :k8s,
+  clusters: %{
+    default: %{
+      conn: "~/.kube/config"
+    }
+  }
+```
+3. If RBAC is enabled, you must have permissions for creating and modifying `CustomResourceDefinition`, `ClusterRole`, `ClusterRoleBinding` and `ServiceAccount`.
+4. Have run generated your manifest `mix bonny.gen.manifest` and installed it using kubectl `kubectl apply -f manifest.yaml`
```

Comment:
“Generate a manifest” might read better. 


Looks good. Much appreciated!

Reply:
Oh wow - I wonder what i was smoking when i wrote this